Model/Transaction.php

```diff
@@ -114,17 +114,64 @@ public function getChildTransactions()
     }
 
     /**
-     * Returns true if transaction's payment method is "wiretransfer"
+     * Returns true if transaction's xml payment method is "wiretransfer"
      *
      * @return bool
      *
      * @since 1.3.0
      */
     public function isPoiPiaPaymentMethod()
+    {
+        return self::_getTransactionXmlPaymentMethod() === BasePoiPiaPaymentMethod::PAYMENT_METHOD_WIRETRANSFER;
+    }
+
+    /**
+     * Returns payment method from transaction xml object
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    private function _getTransactionXmlPaymentMethod()
```

Comment:
Please **postfix with "Name"** as it is returning the payment method name.

Reply:
I put postfix "id", because it technically is a payment method id in xml.